package.json

```diff
@@ -1,26 +1,27 @@
 {
   "name": "@evolv/client",
   "version": "2.1.0",
-  "description": "This is the official JavaScript client for the Evolv Technologies optimization platform.",
-  "main": "dist/index.js",
-  "module": "src/index.js",
-  "types": "src/types.d.ts",
+  "description": "The official JavaScript client for the Evolv AI optimization platform.",
+  "license": "Apache-2.0",
+  "type": "module",
+  "main": "dist/browser/index.cjs",
+  "module": "dist/browser/index.mjs",
+  "types": "dist/types.d.ts",
   "exports": {
-    "import": "./src/index.js",
-    "require": "./dist/index.js"
+    "import": "./dist/browser/index.mjs",
+    "require": "./dist/browser/index.cjs",
+    "node": "./dist/node/index.mjs"
   },
-  "type": "module",
   "homepage": "https://www.evolv.ai",
   "bugs": {
     "url": "https://github.com/evolv-ai/javascript-sdk/issues",
     "email": "support@evolv.ai"
   },
   "files": [
-    "dist/**/*",
-    "src/**/*"
+    "dist/**/*"
   ],
   "directories": {
-    "doc": "./docs"
+    "doc": "./dist/docs"
```

Comment:
Changes autogenerated docs location?

Reply:
I think this reference was just wrong. Docs were already being outputted to _dist/docs_.